### DISCUSSION TITLE
Modifica file YAML tramite VSC - errore salvataggio

### DISCUSSION THREAD

**Discussion by reiner348** [indomus/forum#1794] [HUB: Home Assistant]
Ciao a tutti,
ho un problema con cui mi sto scontrando da un po'... 
Premessa: per modificare i file YAML uso VSC da remoto, con HA installato su raspbian tramite docker. Non ho mai avuto problemi nei mesi scorsi, ultimamente invece non riesco a salvare... Non sempre, a volte, a caso, e una volta che mi compare la prima volta l'avviso, quel file non è più salvabile, anche spegnendo, riavviando, stoppando e riavviando il docker... Niente. Invece altri file riesco a modificarli, anche dopo che uno si blocca. Per esempio, mi si è bloccato il salvataggio di scripts.yaml, ma posso modificare e salvare sensors.yaml.
Ho provato anche a caricare un backup del server su un'altra SD, nel caso fosse un problema di scrittura su SD danneggiata, ma niente, stesso problema...

L'avviso è questo:
Failed to save 'scripts.yaml': Unable to write file 'vscode-remote://ssh-remote+10.0.0.100/home/pi/homeassistant/scripts.yaml' (NoPermissions (FileSystemError): Error: EACCES: permission denied, open '/home/pi/homeassistant/scripts.yaml')

**Comment by snakuzzo** [comment]
1. docker non c'entra niente
2. avrai fatto cavolate con i permessi sui file

**Reply by reiner348** [reply]
Effettivamente sono andato dentro raspbian, nella cartella dei file e, per qualche motivo, scripts era diverso come permessi di modifica... L'ho messo "anyone" ed è andato. Non mi spiego il cambiamento però, non ho mai fatto niente del genere... Va beh, per ora ho risolto. Devo solo trovare il file lovelace che è l'altro che mi dà problemi
